modules/monitoring-silencing-alerts.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// monitoring/configuring-the-monitoring-stack.adoc
+
+[id='monitoring-silencing-alerts-{context}']
+= Silencing alerts
+
+You can either silence a specific alert or silence alerts that match a specification that you define.
+
+.Procedure
+
+To silence a set of alerts by creating an alert specification:
+
+. Navigate to the *Monitoring* -> *Silences* page of the {product-title} web console.
+
+. Click *Create Silence*.
+
+. Populate the *Create Silence* form.
+
+. To create the silence, click *Create*.
+
+To silence a specific alert:
+
+. Navigate to the *Monitoring* -> *Alerts* page of the {product-title} web console.
+
+. For the alert that you want to silence, click the button in the last column and click *Silence Alert*. The *Create Silence* form will appear with prepopulated specification of the chosen alert.
```

Comment:
another way to silence alert, go to alert details page, and click "Silence Alert" from "Actions"